Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -2110,6 +2099,34 @@ public async Task LastMessageTimestamp()
             }
         }
 
+        [FactOnlyTurnAvailable(Timeout = Timeout)]
+        public async Task Restart()
+        {
+            Swarm<DumbAction> swarm1 = CreateSwarm(
+                blockChain: _blockchains[0],
+                iceServers: FactOnlyTurnAvailableAttribute.IceServers
+            );
+            Swarm<DumbAction> swarm2 = _swarms[0];
+
+            try
+            {
+                await StartAsync(swarm1);
+                await BootstrapAsync(swarm2, swarm1.AsPeer);
+                await StartAsync(swarm2);
+                await Task.Delay(1000);
+                await swarm1.StopAsync();
+                Assert.False(swarm1.Running);
+                await Task.Delay(1000);
+                await StartAsync(swarm1);
+                Assert.True(swarm1.Running);
```

Comment:
Can we check some features that swarm provides? (ex. sending/receiving messages)

Reply:
Added. https://github.com/planetarium/libplanet/pull/965/commits/ea93313c69fdcb1f79d5d432767c8956c6f3d1b1